SQL/New_patches/2021-02-19_electrophysiology_annotation_tables.sql

```diff
@@ -1,5 +1,10 @@
 -- SQL tables for BIDS derivative file structure
-
+DROP TABLE IF EXISTS `physiological_annotation_instance`;
+DROP TABLE IF EXISTS `physiological_annotation_parameter`;
+DROP TABLE IF EXISTS `physiological_annotation_archive`;
```

Comment:
These drop tables should not be in patches for existing projects. It adds an unnecessary risk of data loss.